thoth/storages/sync.py

```diff
@@ -548,6 +550,69 @@ def sync_inspection_documents(
 
     return processed, synced, skipped, failed
 
+def sync_security_indicators_documents(
+    document_ids: Optional[List[str]] = None,
+    force: bool = False,
+    graceful: bool = False,
+    graph: Optional[GraphDatabase] = None,
+    is_local: bool = False,
+) -> tuple:
+    """Sync security indicators results into graph."""
+    if is_local and not document_ids:
+        raise ValueError(
+            "Cannot sync documents from local directory without explicitly specifying a list of documents to be synced"
+        )
+
+    if not graph:
+        graph = GraphDatabase()
+        graph.connect()
+
+    if not is_local:
+        security_store = SecurityIndicatorsResultsStore()
+        security_store.connect()
+
+    processed, synced, skipped, failed = 0, 0, 0, 0
+    for security_indicator_key in document_ids or security_store.get_listing():
+        security_indicator_id = security_indicator_key
+
+        if not is_local:
+            security_indicator_id = security_indicator_key.split("/")[0]
+
+        if is_local or "aggregated" in security_indicator_key:
+            si_aggregated_document_id = f"{security_indicator_id}/aggregated"
+            processed += 1
+
+            if force or not graph.si_aggregated_document_id_exist(os.path.basename(security_indicator_id)):
+                try:
+                    if is_local:
+                        # Adjust local
+                        _LOGGER.debug("Loading document from a local file: %r", si_aggregated_document_id)
+                        with open(si_aggregated_document_id, "r") as document_file:
+                            aggregated_document = json.loads(document_file.read())
+                    else:
+                        _LOGGER.info(security_indicator_id)
```

Comment:
This will not be that informative in logs. I think it can be removed, the info message below states what's going on.